app/models/policies/international_relocation_payments/eligibility.rb

```diff
@@ -3,6 +3,15 @@ module InternationalRelocationPayments
     class Eligibility < ApplicationRecord
       self.table_name = "international_relocation_payments_eligibilities"
 
+      PRE_ACADEMIC_YEAR_WINDOW_LIMIT = 6.months
+
+      def self.earliest_eligible_contract_start_date
+        Journeys::GetATeacherRelocationPayment
+          .configuration
+          .current_academic_year
+          .start_of_autumn_term - PRE_ACADEMIC_YEAR_WINDOW_LIMIT
+      end
```

Comment:
I think this method could probably live in the eligibility checker class more comfortably. For context, I think we need to change the name of these `Eligibility` classes to something like `SupportingInformation` - or move their data into the `Claim` itself.

Reply:
Makes sense, have moved it to the checker class 👍